src/Commands/ListAddListCommand.ts

```diff
@@ -0,0 +1,31 @@
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { NativeCallCommand } from "./NativeCallCommand";
+import { Parameter } from "./Parameters/Parameter";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command for a list pop statement.
+ */
+export class ListAddListCommand extends NativeCallCommand {
+    /**
+     * Information on parameters this command takes in.
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("name", "The name of the list.", true),
+        new SingleParameter("name", "The name of the list.", true)
+    ];
+
+    /**
+     * @returns Information on parameters this command takes in.
+     */
+    public getParameters(): Parameter[] {
+        return ListAddListCommand.parameters;
+    }
+
+    /**
+     * @returns Metadata on how to perform the native call. 
+     */
+    protected retrieveNativeCallProperties(): NativeCallProperties {
+        return this.language.properties.lists.concat;
+    }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/3335181/26855186-7b5be5aa-4ace-11e7-8599-1d6d67f6b103.png)

This means you don't have a blank line at the end of the file. Please do.